src/GoogleAnalyticsExtension.php

```diff
@@ -0,0 +1,172 @@
+<?php
+// Google Analytics extension for Bolt
+
+namespace Bolt\Extension\Bolt\GoogleAnalytics;
+
+use Bolt\Asset\Snippet\Snippet;
+use Bolt\Asset\Target;
+use Bolt\Controller\Zone;
+use Bolt\Extension\SimpleExtension;
+use Bolt\Menu\MenuEntry;
+use Silex\ControllerCollection;
+use Bolt\Translation\Translator as Trans;
+use Silex\Application;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Translation\Loader as TranslationLoader;
+
+class GoogleAnalyticsExtension extends SimpleExtension
+{
+
+    protected function registerServices(Application $app)
+    {
+        //Register translations in the service provider.
+        $translationDirectory = $this->getBaseDirectory()->getDir('translations');
+        if ($translationDirectory->exists()) {
+            foreach ($translationDirectory->getContents(true) as $fileInfo) {
+                if ($fileInfo->isFile()) {
+                    list($domain, $extension) = explode('.', $fileInfo->getFilename());
+
+                    $path = $app['resources']->getPath('extensions' . DIRECTORY_SEPARATOR . $fileInfo->getPath());
```

Comment:
For reference, you can just `$path = $app['resources']->getPath('extensions/'  . $fileInfo->getPath());` as the path will be normalised in the resource manager.